scripts/license_check.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+FILES=$(find . -type f -name "*.go"|grep -v 'third_party'|grep -v 'testdata')
```

Comment:
We may want to expand `FILES` to contain more than .go files. Maybe just listing the files of the prpject, besides third_party, testdata or others. Because `scripts/license_check.sh` is missing a license header, and it's not being detected.

Reply:
We might want, I think using only .go atm is sufficient. Also I do not see a point in adding license data in this trivial script. It does not contain anything worthy like business logic etc.